invent-code-0/main.cpp

```diff
@@ -3,15 +3,16 @@
 using namespace std;
 
 int main() {
-  int price_avocado;
-  int price_tomato;
-  int num_avocados;
-  int num_tomatoes;
+  int price_avocado = 3;
+  int price_tomato = 1;
+  int num_avocados = 3;
+  int num_tomatoes = 2;
   // Step 1: Prompt the user to input the number of tomatoes and avocados required.
 
```

Comment:
cout << "How many tomatoes?" << endl;
cin >> num_tomatoes;

This is how I would write a prompt for a user's input to assign a value to the variables. Currently your variable values are static, so you would have to edit the code to change them, rather than run the program again.